exercises/object-oriented-programming/src/main/java/com/cbfacademy/cars/Cars.java

```diff
@@ -0,0 +1,70 @@
+package com.cbfacademy.cars;
+
+public class Cars {
```

Comment:
Class names should be singular unless they're actually representing a plural concept. An instance of this class represents a single car, so it should be named `Car`.